app/views/welcome/inde.html.erb

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Welcome to GDM Budget_Smart</title>
+  <style>
```

Comment:
- [ ] Let's create a CSS file in the assets folder/stylesheets and put out CSS code there. Inline styling is always discouraged.